### PR TITLE
RT-162, RT-189: Refactor views as per Go-Live requirements

### PR DESCRIPTION
**Pull request by dev-digital**
**Description**

This PR brings in changes for RT-162, RT-189 to address presentation  for _Go-Live_ 

###### Note
As it hides some of the _In-Prog_ features which would eventually implemented, code (source and their tests)  have been commented out for easier retrieval. Once work resumes, they can be reverted as required and copies to be deleted.

**Changes**
- Save _In-Prog_ source and test files, Update cypress config accordingly
- Update task-list footer section with _tentative_ content
- Convert Accordion within Supply Chain Summary view to a table w/o action items(editable links)
- Peg Python runtime to _3.8.8_ within GH actions resolve dependency issues
- Address Djagno warnings expected from V 3.2 onwards

**Screenshots**

##### Task-list page

image

##### SC Summary

image

##### Strategic Action Summary

image

### CODE REVIEW

File: defend_data_capture/supply_chains/templates/strategic_action_summary_saved.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      <a class="govuk-breadcrumbs__link" href="{% url 'strat_action_summary' supply_chain.slug %}" aria-current="page">Strategic actions for {{ supply_chain.name }}</a>
+    </li>
+  </ol>
+</nav>
+{% endblock %}
+{% block body %}
+<h1 class="govuk-heading-xl">Strategic actions for
+  <br>
+  {{ supply_chain.name }}</h1>
+<p class="govuk-body">Select a strategic action to view and edit its details.</p>
```
Comment (mforner13) [inline defend_data_capture/supply_chains/templates/strategic_action_summary_saved.html:21]: I reckon we should just say `to view its details` whilst they don't have the ability to edit :)

**Review by mforner13 — COMMENTED** [review]
What's the plan for the text on the task list page? Would you wait for content before merging? :)

File: cypress/integration/sc_summary_spec.js.saved [hunk 2]
```diff
@@ -0,0 +1,125 @@
+import govDepartments from '../fixtures/govDepartment.json'
```
[the remark this answers is not in the document]
Comment (dev-digital) [reply cypress/integration/sc_summary_spec.js.saved:1]: This commit adds 6 files to existing collection of 21(templates and specs).

As we are doing something(cosmetic change) at this end to be un-done at the other end, I was trying to reduce wasted effort. 

I have removed those files now.

**Comment by dev-digital** [comment]
> What's the plan for the text on the task list page? Would you wait for content before merging? :)

As the changes reflect whats in design, I would proceed with merging if this PR gets approved. Then other ticket within RT-162 would address any deviations.

**Review by mforner13 — APPROVED** [review]
Looks good 😄

**Review by NickFitz — APPROVED** [review]
👍

**Pull request merged**